### PR TITLE
Ot147 88

### PR DESCRIPTION
**Pull request by Lucasfranco1**
**SUMARY**

- Added Documentation for Swagger in TestimonialController
- Fixes in class DocumentationMessages in Testimonial 
- Fixes in getTestimonialPages adding defaultValue="0"

### CODE REVIEW

**Comment by JonathanBolatti** [comment]
nice job! I like how you implemented the constants of the server responses

**Review by anavduo — COMMENTED** [review]
Line 91 may be: List of Testimonials in pagination by 10 testimonies, USER and ADMIN

File: src/main/java/com/alkemy/ong/util/DocumentationMessages.java [hunk 1]
```diff
@@ -86,13 +86,13 @@
 
     public static final String TESTIMONIAL_CONTROLLER_DESCRIPTION = "Contains the endpoints of testimonial entity";
 
-    public static final String TESTIMONIAL_CONTROLLER_SUMMARY_CREATE = "Create testimonial";
+    public static final String TESTIMONIAL_CONTROLLER_SUMMARY_CREATE = "Create testimonial, ADMIN";
 
-    public static final String TESTIMONIAL_CONTROLLER_SUMMARY_LIST = "List of Categories pageds";
+    public static final String TESTIMONIAL_CONTROLLER_SUMMARY_LIST = "List of Testimonials in pagination of teen, USER and ADMIN";
```
Comment (pablofrankenn) [inline src/main/java/com/alkemy/ong/util/DocumentationMessages.java:91]: Ten, not teen. : )

**Comment by Lucasfranco1** [comment]
> Line 91 may be: List of Testimonials in pagination by 10 testimonies, USER and ADMIN

Thanks! Done

**Comment by anavduo** [comment]
Nice job!

**Pull request merged**